src/cdk/wizards/previewStateMachineCDKWizard.ts

```diff
@@ -0,0 +1,233 @@
+    private cdkApplication?: CdkAppLocationPickItem
+    private topLevelNode?: TopLevelNodePickItem
+    private stateMachine?: ConstructNodePickItem
+    private promptUser: typeof picker.promptUser
+
+    public constructor(promptUser?: typeof picker.promptUser) {
+        super()
+        this.promptUser = promptUser || picker.promptUser.bind(picker)
+    }
+
+    protected get startStep() {
+        return this.SELECT_WORKSPACE_ACTION
+    }
+
+    private readonly SELECT_WORKSPACE_ACTION: WizardStep = async () => {
+        const cdkAppLocationsHelper: CdkAppLocation[] = await detectCdkProjects(vscode.workspace.workspaceFolders)
+        const cdkAppLocations: CdkAppLocationPickItem[] = cdkAppLocationsHelper.map(obj => {
+            return {
+                label: getCDKAppWorkspaceName(obj.cdkJsonPath),
+                cdkApplocation: obj,
+            }
+        })
+
+        if (cdkAppLocations.length === 0) {
+            cdkAppLocations.push({
+                label: localize('Aws.cdk.app.noWorkspace', '[No workspace found]'),
+                cdkApplocation: undefined,
+            })
+        }
+
+        const quickPick = picker.createQuickPick<CdkAppLocationPickItem>({
+            options: {
+                ignoreFocusOut: true,
+                title: localize('AWS.message.prompt.selectCDKWorkspace.placeholder', 'Select CDK workspace'),
+                step: 1,
+                totalSteps: 3,
+            },
+            buttons: [vscode.QuickInputButtons.Back],
+            items: cdkAppLocations,
+        })
+
+        const choices = await this.promptUser({
+            picker: quickPick,
+            onDidTriggerButton: (button, resolve) => {
+                if (button === vscode.QuickInputButtons.Back) {
+                    resolve(undefined)
+                }
+            },
+        })
+
+        this.cdkApplication = picker.verifySinglePickerOutput<CdkAppLocationPickItem>(choices)
+        return this.cdkApplication ? wizardContinue(this.SELECT_APPLICATION_ACTION) : WIZARD_GOBACK
+    }
+
+    private readonly SELECT_APPLICATION_ACTION: WizardStep = async () => {
+        const appLocation = this.cdkApplication ? this.cdkApplication.cdkApplocation : undefined
+
+        if (!appLocation) {
+            return WIZARD_GOBACK
+        }
+
+        const appNode = new AppNode(appLocation)
+        const constructNodes = await appNode.getChildren()
+        const cdkApplications: TopLevelNodePickItem[] = constructNodes.map(node => {
+            return {
+                label: node.label || '',
+                topLevelNode: node as ConstructNode,
+            }
+        })
+
+        if (cdkApplications.length === 0) {
+            cdkApplications.push({
+                label: localize(
+                    'AWS.cdk.explorerNode.noApps',
+                    `[No cdk application(s) found in workspace '${getCDKAppWorkspaceName(appLocation.cdkJsonPath)}']`
```

Comment:
For localize calls, you cannot use template literals. Instead, replace the substitution with `{0}`, then add another argument to the function call with `getCDKAppWorkspaceName(appLocation.cdkJsonPath)`:
```js
label: localize(
   'AWS.cdk.explorerNode.noApps',
   "[No cdk application(s) found in workspace '{0}']",
   getCDKAppWorkspaceName(appLocation.cdkJsonPath)
)
```